docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -112,7 +112,57 @@ quarkus.hibernate-orm.database.generation=drop-and-create
 Note that these configuration properties are not the same ones as in your typical Hibernate Reactive configuration file.
 They will often map to Hibernate Reactive configuration properties but could have different names and don't necessarily map 1:1 to each other.
 
-Also, Quarkus will set many Hibernate Reactive configuration settings automatically, and will often use more modern defaults.
+Blocking (non-reactive) and reactive configuration can be mixed together in the same project.
+
+[[orm-and-reactive-extension-simultaneously]]
+=== Hibernate ORM and Reactive extensions simultaneously
+
+If you add both Hibernate ORM and Hibernate Reactive extensions to your Quarkus app, they can work together.
+
+This is useful if your app normally uses Hibernate ORM (which is blocking), but you want to try Hibernate Reactive to see if it works better for your case.
+
+By adding the second extension, you can use the reactive API in another part of your code - without needing to create a separate app.
+
+* To use the both extension simultaneously, add both extension to the `pom.xml` file:
++
+[source,xml]
+----
+        <!-- Hibernate reactive -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-reactive</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-reactive-pg-client</artifactId>
+        </dependency>
+
+        <!-- Hibernate ORM -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-postgresql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-orm</artifactId>
+        </dependency>
+----
+
+* Also update the `applications.properties` file:
+[source,properties]
+----
+%prod.quarkus.datasource.reactive.url=postgresql:///your_database
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/hibernate_orm_test
+----
+
+* The presence of the JDBC driver will trigger Hibernate ORM. If you want to disabled it, you can use:
++
+[source,properties]
+----
+quarkus.hibernate-orm.blocking=false
+----
+
+Quarkus will set many Hibernate Reactive configuration settings automatically, and will often use more modern defaults.
 
 WARNING: Configuring Hibernate Reactive using the standard `persistence.xml` configuration file is not supported.
 
```

Comment:
This (and the text below) is now part of the section "Hibernate ORM and Reactive extensions simultaneously", which I guess was not the point?

Maybe move this section just before "CDI integration", and insert a link to it in the first line "Blocking (non-reactive) and reactive configuration can be mixed together in the same project."?

Reply:
Done https://github.com/quarkusio/quarkus/pull/44473/commits/df01abf3672fff0ff602ee0d8b0da46ea1f3a34b